docs/intel/what-is-peacock.md

```diff
@@ -7,21 +7,23 @@ Peacock is essentially a Hitman server emulator that runs on your own PC.
 
 When you start the game, it doesn't connect to IOI's servers, only Peacock.
 
-## Benefits
+## How does Peacock Work?
 
-The benefits of using Peacock include:
+Here's a brief breakdown of what Peacock does:
 
--   Access Hitman's rating system without being connected to IOI's servers
+-   Peacock lets you access Hitman's rating system without being connected to IOI's servers.
     -   **Note: Peacock is not a fully offline experience.**
     -   Each time you start the game, you will initially need to be connected to the internet still, to pass Steam or Epic verification.
     -   After you have passed Steam / Epic verification and have connected to Peacock, you are free to disconnect your internet if you like and you will stay connected to Peacock for the duration of your game session.
--   Obtain unlocks without grinding mastery levels
-    -   You will only have access to DLC items if you have purchased said DLC. Piracy is not condoned.
-    -   **Unlocking items as you go is supported**. Nevertheless it is configurable and you can opt-out of it by editing `options.ini`.
-    -   Challenges is mostly implemented, but some challenges might not be working. If you find one, please create an issue to report this bug.
+-   **In general, as of Peacock v6.31, challenges, mastery, progression, scoring, earning merces (Freelancer) are all supported, and the experience is 1:1 with the official servers experience**
+    -   Minor Exception: Some challenges, especially for Freelancer, may have some minor bugs. Please report any issues to the Peacock Discord `#help` channel (https://thepeacockproject.org/discord).
+    -   Please check the `options.ini` file in your Peacock install folder for any additional configuration tweaks, such as **turning progression / challenge completion on and off** (e.g. if you want to get all unlocks right away on purpose).
+    -   **By default, progression / challenge completion is turned on to simulate an official servers experience.**
     -   If it is your first time playing (on either Peacock or official servers), some exits will be locked. You will need to unlock Dubai exits ([need to unlock the elevators](https://youtu.be/IEQgRQyQRf8)), Dartmoor exits ([need to exit via bike at least once](https://youtu.be/AJtJZe9jEi8?t=151)), and Colorado exits ([need to use tornado exit at least once](https://youtu.be/3XKWHrKpXwk?t=140)). These exits are handled client-side.
--   Play [custom](../custom-content.md) or otherwise discontinued / time limited content such as [Ghost Mode](../ghost-mode), legacy escalations, elusive targets.
-    -   Display leaderboards for other Peacock users
+-   Peacock lets you play [custom](../custom-content.md) or otherwise discontinued / time limited content such as [Ghost Mode](../ghost-mode), escalations, elusive targets, and contracts.
```

Comment:
Note that Ghost Mode only works on Steam. Add Sniper Assassin multiplayer to this too (with the same Steam restriction). It's worth noting that it's supported on H2/3 (with a mod required for GM to work on Hitman 3).